lib/logstash/outputs/lumberjack.rb

```diff
@@ -58,10 +59,10 @@ def register
   public
   def receive(event)
     return unless output?(event)
-    return if event == LogStash::SHUTDOWN
```

Comment:
this is strange, by removing this return it means the special SHUTDOWN event will be sent to the codec. is that intended?